src/js/renderers/NavigatorLayout.js

```diff
@@ -23,7 +24,7 @@ import {
 } from '@wordpress/components';
 
 import { NavigatorLayoutRenderer } from "./NavigatorRenderer";
-import { ArrayControlRenderer } from "./ArrayLayoutRenderer";
+import ArrayControlRenderer from "./ArrayLayoutRenderer";
```

Comment:
This array renderer after proceed with HOC `withJsonFormsArrayControlProps` now can react to data change
Note that this is displayed on NavigatorScreen as form fields